aim/web/ui/src/config/enums/tableEnums.ts

```diff
@@ -1,7 +1,7 @@
 enum RowHeightEnum {
-  small = 28,
-  medium = 32,
-  large = 36,
+  small = 22,
```

Comment:
What about changing the small height to 21, like in google sheets

Reply:
21 is odd number it doesn't go well with other tokens of our system